### PR TITLE
Create release-03 presubmits for Kueue

### PR DESCRIPTION
**Pull request by ArangoGutierrez**

### CODE REVIEW

**Comment by tenzen-y** [comment]
/hold

We are considering bumping the Go version on the release-0.3 branch.

/cc @alculquicondor

File: config/jobs/kubernetes-sigs/kueue/kueue-presubmits-release-0-1.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+presubmits:
```
Comment (alculquicondor) [inline config/jobs/kubernetes-sigs/kueue/kueue-presubmits-release-0-1.yaml:1]: what about the rest of the presubmits?

Comment (ArangoGutierrez) [reply config/jobs/kubernetes-sigs/kueue/kueue-presubmits-release-0-1.yaml:1]: do we want to for a N-2 release?

File: config/jobs/kubernetes-sigs/kueue/kueue-presubmits-main.yaml [hunk 2]
```diff
@@ -1,6 +1,8 @@
 presubmits:
   kubernetes-sigs/kueue:
   - name: pull-kueue-test-unit-main
+    branches:
+    - ^main
```
[the remark this answers is not in the document]
Comment (ArangoGutierrez) [reply config/jobs/kubernetes-sigs/kueue/kueue-presubmits-main.yaml:5]: Example https://github.com/kubernetes/test-infra/tree/master/config/jobs/kubernetes-sigs/cluster-api 

projects tend to do this, it gives more control over what to test per release, understanding that some release deps might change, skipping branches is hard, for this same example reason, the go version. is better to maintain a file per branch moving forward .

[the remark this answers is not in the document]
Comment (alculquicondor) [reply config/jobs/kubernetes-sigs/kueue/kueue-presubmits-main.yaml:5]: Sounds good, please add it to the release issue template

File: config/jobs/kubernetes-sigs/kueue/kueue-presubmits-release-0-1.yaml (hunk 1 above)
Comment (ArangoGutierrez) [reply config/jobs/kubernetes-sigs/kueue/kueue-presubmits-release-0-1.yaml:1]: fine by me, unless we plan to have support for 0.1 today ;P  jejeje 
to that point, do we state in the docs about support policy (N-1 stuff)

Comment (alculquicondor) [reply config/jobs/kubernetes-sigs/kueue/kueue-presubmits-release-0-1.yaml:1]: We don't.
Unofficially, we just plan to support 0.3+, unless someone comes knocking the door for a fix in 0.2.
But 0.1 is old enough that I don't see it happening.

**Comment by alculquicondor** [comment]
Do we need to remove `always_run: true`? But then it wouldn't be required for the specific branches?
Did we miss anything else?

**Comment by ArangoGutierrez** [comment]
> Do we need to remove `always_run: true`? But then it wouldn't be required for the specific branches? Did we miss anything else?

You are right, see https://github.com/kubernetes/test-infra/pull/29364

**Pull request merged**